docs/gestor_de_tareas.md

```diff
@@ -0,0 +1,32 @@
+# Gestor de Tareas
+
+A continuación comento algunos gestores de tareas que se consideran para el proyecto y algunos pros y contras para decidirnos por uno de ellos:
+
+- [Gulp](https://gulpjs.com/docs/en/getting-started/quick-start): Gestor de tareas que te permite [usar tu conocimiento de Javascript](https://gulpjs.com/docs/en/getting-started/javascript-and-gulpfiles) para escribir tareas automatizables que se repiten en tu proyecto y que terminará ejecutando con **gulp <nombre-tarea>**
```

Comment:
gulp no sirve para go. Por favor, no pongáis gestores de tareas rándom, sino los que sean realmente relevantes.

Reply:
Pondré sólo los de Go.

Reply:
No te estoy diciendo que pongas *sólo* los de Go. Te estoy diciendo que establezcas un criterio para buscar gestores de tareas (incluyendo mejores prácticas) y pongas sólo los que satisfagan ese criterio.